test/javascripts/acceptance/composer-test.js

```diff
@@ -720,8 +720,6 @@ QUnit.test("Loading draft also replaces the recipients", async assert => {
 QUnit.test(
   "Deleting the text content of the first post in a private message",
   async assert => {
-    Discourse.SiteSettings.allow_uncategorized_topics = false;
```

Comment:
I imagine false is the default and this is why you removed it ?

Reply:
Oh while refactoring I noticed it was not necessary here. I didn't notice the other places.